### PR TITLE
Updates left TOC to match home page

### PR DESCRIPTION
**Pull request by nbarbettini**
Visual change only (no files were moved or renamed). Aligns the left-side page TOC with the documentation home page.

Still to be done (waiting on some tech fixes):
- Updating URLs and creating appropriate redirects
- Updating titles to match conventions for each category

### CODE REVIEW

**Comment by nbarbettini** [comment]
Screenshot:

image

File: packages/@okta/vuepress-site/.vuepress/nav/main.js [hunk 1]
```diff
@@ -29,25 +41,18 @@ module.exports = [
...
+      { title: 'Add User Consent to Your Authentication Flow', link: '/docs/how-to/add-user-consent-to-flow/'},
+      { title: 'Upload Your Own Certificates for Outbound SAML Apps', link: '/docs/how-to/byo_saml/'},
+      { title: 'Create an ID Token or Access Token Containing a Groups Claim', link: '/docs/how-to/creating-token-with-groups-claim/'},
+      { title: 'Share Application Key Credentials for IdPs Across Apps', link: '/docs/how-to/sharing-cert/'},
+      { title: 'Upgrade SAML Apps to SHA256', link: '/docs/how-to/updating_saml_cert/'}
     ]
   },
   {
-    title: 'API Reference',
+    title: "Reference",
```
Comment (swiftone) [inline packages/@okta/vuepress-site/.vuepress/nav/main.js]: Note that the main nav "Docs" still has "API Reference"

Comment (alexdahl-okta) [reply packages/@okta/vuepress-site/.vuepress/nav/main.js]: Good catch. Indeed, the global nav dropdown and Reference section index should also be updated to "Reference". But, no opinion from me about whether that should happen in this PR or separately.

reference

**Comment by nbarbettini** [comment]
Fixed: 

image

Although parts of the site owned by Drupal aren't covered by this fix, so it will be inconsistent until that is fixed as well. I'll file the ticket.

**Pull request merged**